waspc/cli/Command/Db/Migrate.hs

```diff
@@ -86,3 +94,39 @@ applyAvailableMigrationsAndGenerateClient genProjectRootDir = do
                                genClientError
         Right () -> waspSaysC "Done."
 
+
+data MigrationDirCopyDirection = CopyMigDirUp | CopyMigDirDown deriving (Eq)
+
+-- | Copy migrations directory between Wasp source and the generated project.
+copyDbMigrationsDir
+    :: MigrationDirCopyDirection -- ^ Copy direction (source -> gen or gen-> source)
```

Comment:
It is slightly suspicious practice to pass a flag that determines functions behavior (Uncle Bob mentioned that I believe, and we also don't see it often in practice). It indicates that two different functions are crammed into one. Instead, it would probably be better to have two functions, and then you can isolate common part(s) into a third functions that these two call.
I am ok with leaving it like this here since this will likely change soon anyway and it is localized, but wanted to mention still.